build/Build.Pack.cs

```diff
@@ -454,4 +476,37 @@ void PackTarballs(string runtimeId)
             ArtifactsDirectory / "zip",
             $"tentacle-{FullSemVer}-{NetCore}-{runtimeId}.tar.gz");
     }
+
+    void BuildAndPushOrLoadKubernetesTentacleContainerImage(bool push, bool load)
+    {
+        DockerTasks.DockerBuildxBuild(settings =>
+            settings.AddBuildArg($"BUILD_NUMBER={FullSemVer}", $"BUILD_DATE={DateTime.UtcNow:O}")
+                .SetPlatform("linux/arm64,linux/amd64")
```

Comment:
I've removed linux/arm/v7 for now because I was having some trouble with docker failing to run commands when building the container. I'll take a look later but it's not really too important.